reservation_units/models.py

```diff
@@ -615,10 +620,18 @@ def get_next_reservation(self, end_time: datetime.datetime, reservation=None):
         if reservation:
             qs = qs.exclude(id=reservation.id)
 
-        return qs.order_by("-begin").first()
+        if not include_blocked:
```

Comment:
I'd change this to `if exclude_blocked` => not clauses are kinda harder to read.

Reply:
True! Fixed! 👍